.ci_tools/build_samples.sh

```diff
@@ -23,6 +24,7 @@ declare bundles=(
     samples/MegaDrone/build/outputs/bundle/debug/MegaDrone.aab
     samples/RhythmGame/build/outputs/bundle/ndkExtractorDebug/RhythmGame.aab
     samples/LiveEffect/build/outputs/bundle/debug/LiveEffect.aab
+    apps/OboeTester/app/build/outputs/bundle/debug/app.aab
```

Comment:
Why is OboeTester using "app.aab" instead of OboeTester.aab?
I'm worried that if I change OboeTester build then I might break this.

Reply:
Which file defines the bundle name?

Reply:
the directory "app" defined the apk name, by default ( in apps/OboeTester/app <-- this 'app')